_config.yml

```diff
@@ -301,6 +301,10 @@ component_releases:
 
 release_archive:
     flink:
+      -
+        version_short: 1.8
+        version_long: 1.8.2
```

Comment:
these entries aren't being sorted automatically; since we order the release entries in this list by version this entry should go below 1.9.0 .